shotgun_api3/shotgun.py

```diff
@@ -136,6 +136,12 @@ class ShotgunFileDownloadError(ShotgunError):
     """
     pass
 
+class ShotgunThumbnailNotReady(ShotgunError):
+    """
+    Exception for file download-related errors.
```

Comment:
Do we want to be more specific about what this Exception is for (maybe something like "attempting to download or use a pending thumbnail") or do you see it as being used more generally as well down the road?

Reply:
Good catch! I totally forgot to update the description. Done!